Resources/doc/first_steps.rst

```diff
@@ -115,9 +115,8 @@ Persisting Objects to MongoDB
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Now that you have a mapped ``Product`` document complete with getter and
-setter methods, you're ready to persist data to MongoDB. From inside a controller,
-this is pretty easy. Add the following method to the ``DefaultController``
-of the bundle:
+setter methods, you're ready to persist data to MongoDB. Let's try it from inside
+a controller. Create new Controller class inside source directory of your project:
```

Comment:
``DefaultController` was already created in this chapter, please bring back the part about adding a new method to it

Reply:
At the top of the file I've removed section with `bin/console generate:bundle --namespace=Acme/StoreBundle` command from which DefaultController came from

Reply:
Ah sorry then 👍  so we need to just change
```suggestion
a controller. Create a new Controller class inside source directory of your project:
```

Also what do you think about having entire controller in example? Showing only method to add made sense when controller was already there, now it looks a bit off IMO

Reply:
Ok, will add class name to the example. Also the file path inside codeblock in the next section _Fetching Objects from MongoDB_. That example doesn't need full class, right?

Reply:
> That example doesn't need full class, right?

Right :+1: